app/models/georeference.rb

```diff
@@ -93,6 +91,8 @@ class Georeference < ApplicationRecord
   has_many :collection_object_otus, -> { unscope(:order) }, through: :collection_objects, source: 'otu'
   has_many :field_occurrence_otus, -> { unscope(:order) }, through: :field_occurrences, source: 'otu'
 
+  # TODO: this should probably go away, it's misleading now with multiple
+  # throughs possible.
   has_many :otus, through: :collection_objects, source: 'otus'
 
```

Comment:
Try go elminate or rename for a more specific source of the otu, perhaps add a parallel method for the other sources of otus.

Reply:
I'll do a followup on this one (`.otus` might not be an easy association to adjust).